File: C++17/백준/Bronze/26307. Correct/Correct.cc
#include <iostream>
#include <cmath>
using namespace std;

int main (void) {
    int H, M;
    cin >> H >> M;
    
    int result_H = abs(H - 9) * 60;
    
    cout << result_H + M << endl;
    
    return 0;
}

File: Java/백준/Bronze/26307. Correct/Correct.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int H = sc.nextInt();
        int M = sc.nextInt();
        
        int result_H = Math.abs(H - 9) * 60;
        
        System.out.println(result_H + M);
        
        sc.close();
    }
}